ci/edmtool.py

```diff
@@ -234,6 +240,23 @@ def install(runtime, toolkit, pillow, environment, source):
     click.echo('Done install')
 
 
+@cli.command()
+@click.option('--runtime', default='3.6')
+@click.option('--toolkit', default='null')
+@click.option('--pillow', default='pillow')
+@click.option('--environment', default=None)
+def shell(runtime, toolkit, pillow, environment):
+    """ Create a shell into the EDM development environment
+    (aka 'activate' it).
+
+    """
+    parameters = get_parameters(runtime, toolkit, pillow, environment)
+    commands = [
+        "edm shell -e {environment}",
```

Comment:
Not really your fault, but it seems a little bit smelly to split on whitespace when turning these strings into command lines. There are cases where that will fail.